src/core/ext/filters/client_channel/resolver/dns/c_ares/dns_resolver_ares.cc

```diff
@@ -381,9 +382,11 @@ class AresDNSResolver : public DNSResolver {
       GRPC_CLOSURE_INIT(&on_dns_lookup_done_, OnDnsLookupDone, this,
                         grpc_schedule_on_exec_ctx);
       MutexLock lock(&mu_);
+      pollset_set_ = grpc_pollset_set_create();
```

Comment:
Please initialize this in the initializer list instead of in the body of the ctor.

Reply:
Done